app/models/permission.rb

```diff
@@ -6,9 +6,9 @@ class Permission < ApplicationRecord
                            toggle_ready_status destroy_membership destroy_any_card
                            like_card create_comments create_action_items update_action_items
                            destroy_action_items move_action_items close_action_items
-                           complete_action_items reopen_action_items].freeze
+                           complete_action_items reopen_action_items like_comment].freeze
```

Comment:
Please don't add the `like_comment` to the Creator and Member identifiers otherwise they will be able to put like on all comments including their own.
https://github.com/cybergizer-hq/retrospective/blob/lj/add-like-comment-permission/app/operations/boards/build_permissions.rb#L20-L24